### PR TITLE
added avatar component

### PR DESCRIPTION
**Pull request by PeeyushPrashant**

### CODE REVIEW

**Review by MurtazaBagwala07 — APPROVED** [review]
looks good , just add some alt title to images

**Review by ddivyansh18 — APPROVED** [review]
Looks good, nicely organised

File: Components/Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <div class="avatar-container">
+      <img class="avatar sm" src="/asset/image/virat.jpg" alt="" />
+      <img class="avatar md" src="/asset/image/apd.jpg" alt="" />
+      <img class="avatar lg" src="/asset/image/cr7.jpg" alt="" />
+    </div>
+
+    <div class="avatar-container">
+      <img class="avatar square sm" src="/asset/image/virat.jpg" alt="" />
+      <img class="avatar square md" src="/asset/image/apd.jpg" alt="" />
+      <img class="avatar square lg" src="/asset/image/cr7.jpg" alt="" />
```
Comment (anuj-kumary) [inline Components/Avatar/avatar.html]: Always give **alt** (alternate) in (avatar, badge).

Comment (PeeyushPrashant) [reply Components/Avatar/avatar.html]: okay, sure

**Comment by PeeyushPrashant** [comment]
> looks good , just add some alt title to images

okay,sure

**Pull request merged**